package.json

```diff
@@ -43,6 +43,7 @@
     "preview": "vite preview",
     "lint": "prettier --check .",
     "format": "prettier --write .",
+    "pack-build": "npm run build && npm pack",
```

Comment:
I’m not opposed, but I was wondering why you added this script. Are you using it during development? If so, you might find [this section of the README](https://github.com/RevenueCat/purchases-js?tab=readme-ov-file#install-the-library-in-a-local-project) helpful! 😊

Reply:
packing is what I use to embed the library on the WPL PRs. I figured there was no harm to leave it there